adapters/repos/db/lsmkv/bucket.go

```diff
@@ -165,6 +201,27 @@ func NewBucket(ctx context.Context, dir, rootDir string, logger logrus.FieldLogg
 	return b, nil
 }
 
+func (b *Bucket) PropertyPrefix() []byte {
+	return []byte("")
+}
```

Comment:
This satisfies the interface "BucketInterface".  It is constant because buckets don't handle propertprefix, only proxy buckets read that value

Reply:
ok, got it.
so does `BucketInterface` need `PropertyPrefix` method? prefix is internal value of proxy bucket, therefore proxy can access it without public method. where else this prefix could be used?

Reply:
currently this method is used just in `proxy_bucket_test` (but it does not have to be as test have direct access to `BucketProxy::propertyPrefix` field anyway).
Is there a reason to keep this method in BucketInterface? where else it will be used?